monai/transforms/croppad/array.py

```diff
@@ -142,6 +143,27 @@ def _pt_pad(img: torch.Tensor, pad_width, mode, **kwargs) -> torch.Tensor:
         # torch.pad expects `[B, C, H, W, [D]]` shape
         return pad_pt(img.unsqueeze(0), pt_pad_width, mode=mode, **kwargs).squeeze(0)
 
+    @staticmethod
+    def pad_nd(img_t, to_pad_, mode, **kwargs):
```

Comment:
could you please remove the underscore `_`,  `to_pad_` -> `to_pad` and also add some docstring? I can also push some docstring to your PR directly before merging this PR.

Reply:
OK, will do that.